src/App.vue

```diff
@@ -478,13 +627,38 @@ export default {
             ipcRenderer.send('set-vircadia-location');
         },
         checkForUpdates: function() {
+            this.disableUpdateButton = true;
+            this.disableLaunchButton = true;
+            
             const { ipcRenderer } = require('electron');
             ipcRenderer.send('check-for-updates');            
+        },
+        resetDownloadButton: function() {
```

Comment:
```suggestion
        resetDownloadButton: function () {
```
Should have a space between `function` and `(` in anonymous functions, here and throughout file.